### PR TITLE
Main update into Master.

### PR DESCRIPTION
**Pull request by JakeDaBoss**
@BlueNexus @Lykayos

### CODE REVIEW

**Comment by JakeDaBoss** [comment]
Additional info is that this is the stable update into Master, so we can use that instead.

File: code/game/objects/items/weapons/manuals.dm [hunk 1]
```diff
@@ -322,7 +322,7 @@
 
 				<a name='6'><h3>Step 6: Get clean SEs for body</h3>
 				Cloning is a finicky and unreliable process. Whilst it will most certainly bring someone back from the dead, they can have any number of nasty disabilities given to them during the cloning process!
-				For this reason, you need to prepare a clean, defect-free Structural Enzyme (SE) injection for when they're done. If you're a competent Geneticist, you will already have one ready on your working computer.
+				For this reason, you need to prepare a clean, defect-free Structural Enzyme (SE) injection for when they're done. If you're a competent Medical Examiner, you will already have one ready on your working computer.
```
Comment (BlueNexus) [inline code/game/objects/items/weapons/manuals.dm:325]: I fail to see what this has to do with a medical examiner.

File: code/game/objects/effects/decals/posters/bs12.dm [hunk 2]
```diff
@@ -117,7 +117,7 @@
 /datum/poster/bay_24
 	icon_state="bsposter24"
 	name = "Responsible medbay habits, No #259"
-	desc = "A poster with a nervous looking geneticist on it states; \"Friends Tell Friends They're Clones. It can cause severe and irreparable emotional trauma if a person is not properly informed of their recent demise. Always follow your contractual obligation and inform them of their recent rejuvenation.\""
+	desc = "A poster with a nervous looking medical examiner on it states; \"Friends Tell Friends They're Clones. It can cause severe and irreparable emotional trauma if a person is not properly informed of their recent demise. Always follow your contractual obligation and inform them of their recent rejuvenation.\""
```
Comment (BlueNexus) [inline code/game/objects/effects/decals/posters/bs12.dm:120]: Also has nothing to do with medical examiners.

File: code/modules/clothing/under/jobs/medsci.dm [hunk 3]
```diff
@@ -58,8 +58,8 @@
 	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 10, rad = 0)
 
 /obj/item/clothing/under/rank/geneticist
```
Comment (BlueNexus) [inline code/modules/clothing/under/jobs/medsci.dm:60]: Should be changed to rank/medexam

**Review by BlueNexus — CHANGES_REQUESTED** [review]

**Comment by JakeDaBoss** [comment]
I'll work on these later.

**Pull request closed without merging**